pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/export/tabular/ExcelParticipantExporter.java

```diff
@@ -44,21 +46,16 @@ public void export(Response response) throws IOException {
             writeRowToSheet(rowValues, i + 2);
         });
 
-        try (ServletOutputStream servOut = response.raw().getOutputStream()) {
-            workbook.write(servOut);
-        }
-        workbook.dispose();
-        workbook.close();
-
+        writeAndCloseSheet(os);
     }
 
-    private void setResponseHeaders(Response response) {
-        response.type(MediaType.OCTET_STREAM.toString());
-        response.header("Access-Control-Expose-Headers", "Content-Disposition");
-        response.header("Content-Disposition", "attachment;filename=" + getExportFilename(fileFormat));
+    protected void writeAndCloseSheet(OutputStream os) throws IOException {
+        workbook.write(os);
+        workbook.dispose();
+        workbook.close();
```

Comment:
#Optional
You could use so called `try-with-resource` technique which automatically closes any auto closable and catches exceptions
`try (workbook) {
   workbook.write(os);
   workbook.dispose();
}`